platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/OcspService.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.handler.pki;
+
+import java.security.cert.X509Certificate;
+
+public interface OcspService {
```

Comment:
❗️ I think we should make a new API bundle for this service that is separate from the PKI handler. Yes, right now that handler is the only thing that is going to use it, but that may not be true going forward. We would also want to pull the implementation out of the PKI handler as well. This interface also needs the "experimental" comment block.